src/Symfony/Component/String/Resources/bin/update-data.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\String\Data\WcswidthDataGenerator;
+
+require_once __DIR__.'/common.php';
+require_once __DIR__.'/autoload.php';
```

Comment:
let's move everything in a single file, no need for these two ones.
Also this should be moved inside the Resources directory and have the export-ignore flag